Babylon/groups/azure/groups/acr/__init__.py

```diff
@@ -0,0 +1,17 @@
+from click import group
+from click import pass_context
+from click.core import Context
+from .commands import list_commands
+from .groups import list_groups
+
+
+@group()
+@pass_context
```

Comment:
We could add the `require_external_program('docker')` here instead of putting it in each sub-command of the group since all commands in `azure acr` would require the user to have a valid install of `docker`